packages/fleather/lib/src/widgets/editable_text_block.dart

```diff
@@ -318,8 +318,13 @@ class _NumberPoint extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final styleHeight = style.height;
     return Container(
-      alignment: AlignmentDirectional.topEnd,
+      // Empirically, depending on height of the style, we need to
+      // align at top or at bottom
+      alignment: styleHeight == null || styleHeight <= 1
```

Comment:
@Amir-P If you can find anything better, it'll be most welcome
I'm not satisfied with this to be honest